README.md

```diff
@@ -54,6 +54,28 @@ Magento 2
     - Open Setup Klaviyo User from the sidebar
     - Enter the username, password, and email for the credentials
     - Save config
+
+
+## Making updates
+1) Follow Klaviyo's standard process of making a pull request and getting it reviewed before merging.
+2) Update CHANGELOG.md. Details on formatting the changelog (including categorizing changes) can be found here: [keepachangelog.com](https://keepachangelog.com/en/1.0.0/)
+    1) If this is a change that will not immediately get sent along to Magento i.e. not a version update:
+        1) Add any changes under the [`[Unreleased]`](https://github.com/klaviyo/magento2-klaviyo/compare/1.0.1...HEAD) section. This will be a comparision of the most recent commits to the latest tagged version.
```

Comment:
i do not get this whole notion of unreleased, why there going to be PRs approved and not released?

Reply:
this is so you don't need to do a release for every update, Ex. you want to add a couple changes but are separate pr's prior to a release